lobster/related.lobster

```diff
@@ -0,0 +1,88 @@
+
+import std
+import dictionary
+import json
+
+let TOPN = 5
+
+struct Post:
```

Comment:
You made this a `struct`, which should generally be faster than a `class` since it is copied rather than dynamically allocated, but it may actually be worse in `for (posts)` because of the copy.. worth trying both and seeing which works better.